README.md

```diff
@@ -95,7 +95,8 @@ Now, save the passphrase you used in the geth node into a text file called passw
 Build our system first
 
 ```
-bazel build //...
+bazel build //beacon-chain/...
+bazel build //client/...
```

Comment:
Please remove the `...`. In the latter case, it will still build the failing target on mac //client:image. 

We should recommend that the user build the specific binaries:

```
bazel build //beacon-chain:beacon-chain
bazel build //client:client
```